docs/CIRCUIT_JSON_PCB_OVERVIEW.md

```diff
@@ -231,6 +231,13 @@ export interface PcbPlacementError {
   message: string
 }
 
+export interface PcbPadOverlapError {
+  type: "pcb_pad_overlap_error"
+  pcb_error_id: string
+  message: string
+  pad_ids: string[]
+}
+
```

Comment:
make sure to always use fully qualified ids, also i'm proposing a rename because pads or plated holes etc. can overlap

```suggestion
export interface PcbFootprintOverlapError {
  type: "pcb_footprint_overlap_error"
  pcb_error_id: string
  message: string
  pcb_smtpad_ids: string[]
  pcb_plated_hole_ids: string[]
}

```